extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -699,11 +735,14 @@ export class AmpLiveList extends AMP.BaseElement {
    * Iterates over the child elements from first to last and invokes the
    * callback with the current child element passed in as the first argument.
    *
-   * @param {!Element} parent
+   * @param {?Element} parent
    * @param {function(!Element)} cb
    * @private
    */
   eachChildElement_(parent, cb) {
+    if (!parent) {
```

Comment:
it probably makes more sense for it to be handled by the caller. the only reason why these might be null are the values supplied on the call site are all elements that are only ready during buildCallback